src/locale/languages/uk.js

```diff
@@ -0,0 +1,349 @@
+      ownerTitle: 'ВЛАСНИК',
+      contributorsTitle: 'УЧАСНИКИ',
+      noContributorsMessage: 'Учасники не знайдені',
+      sourceTitle: 'КОД',
+      readMe: 'README',
+      viewSource: 'Дивитись код',
+      issuesTitle: 'ЗАДАЧІ',
+      noIssuesMessage: 'Немає задач',
+      noOpenIssuesMessage: 'Немає відкритих задач',
+      viewAllButton: 'Дивитись все',
+      newIssueButton: 'Нова задача',
+      pullRequestTitle: 'PULL-ЗАПИТИ',
+      noPullRequestsMessage: 'Немає pull-запитів',
+      noOpenPullRequestsMessage: 'Немає відкритих pull-запитів',
+      starsTitle: 'Зірок',
+      forksTitle: 'Форків',
+      forkedFromMessage: 'склоновано від',
+      starred: 'Відмічено',
+      watching: 'Спостерігаю',
+      watchers: 'Спостерігачів',
+    },
+    codeList: {
+      title: 'Код',
+    },
+    issueList: {
+      title: 'Задачі',
+      openButton: 'Відкрито',
+      closedButton: 'Закрито',
+      searchingMessage: 'Пошук по {{query}}',
+      noOpenIssues: 'Не знайдено відкритих задач!',
+      noClosedIssues: 'Не знайдено закритих задач!',
+    },
+    pullList: {
+      title: 'Pull-запити',
+      openButton: 'Відкрито',
+      closedButton: 'Закрито',
+      searchingMessage: 'Пошук по {{query}}',
+      noOpenPulls: 'Не знайдено відкритих pull-запитів!',
+      noClosedPulls: 'Не знайдено закритих pull-запитів!',
+    },
+    pullDiff: {
+      title: 'Порівняти зміни',
+      numFilesChanged: '{{numFilesChanged}} файлів',
+      new: 'НОВИЙ',
+      deleted: 'ВИДАЛЕНИЙ',
+      fileRenamed: 'Файл перейменований без будь-яких змін',
+    },
+    readMe: {
+      readMeActions: 'Дії з README',
+      noReadMeFound: 'He вдалось знайти README.md',
+    },
+  },
+  organization: {
+    main: {
+      membersTitle: 'УЧАСНИКИ',
+      descriptionTitle: 'ОПИС',
+    },
+    organizationActions: 'Дії з організацією',
+  },
+  issue: {
+    settings: {
+      title: 'Налаштування',
+      pullRequestType: 'Pull-запити',
+      issueType: 'задачу',
+      applyLabelButton: 'Додати ярлик',
+      noneMessage: 'Поки немає',
+      labelsTitle: 'ЯРЛИКИ',
+      assignYourselfButton: 'Призначити самому собі',
+      assigneesTitle: 'ВІДПОВІДАЛЬНІ',
+      actionsTitle: 'ДІЇ',
+      unlockIssue: 'Розблокувати {{issueType}}',
+      lockIssue: 'Блокувати {{issueType}}',
+      closeIssue: 'Закрити {{issueType}}',
+      reopenIssue: 'Відкрити знову {{issueType}}',
+      areYouSurePrompt: 'Ви впевнені?',
+      applyLabelTitle: 'Додати мітку до цієї задачі',
+    },
+    comment: {
+      commentActions: 'Дії З Коментарем',
+      editCommentTitle: 'Редагувати Коментар',
+      editAction: 'Редагувати',
+      deleteAction: 'Видалити',
+    },
+    main: {
+      assignees: 'Відповідальні',
+      mergeButton: 'Приняти pull-запит',
+      noDescription: 'Немає опису.',
+      lockedCommentInput:
+        'Заблоковано, але ви все ще можете прокоментувати...',
+      commentInput: 'Додати коментар...',
+      commentButton: 'Відправити',
```

Comment:
Please delete `commentButton`. Sorry I didn't notice this one before.

Reply:
Resolved